src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts

```diff
@@ -11,7 +12,19 @@ interface Status {
   templateUrl: './candidate-card-popup.component.html',
   styleUrls: ['./candidate-card-popup.component.scss'],
 })
-export class CandidateCardPopupComponent {
+export class CandidateCardPopupComponent implements OnInit {
+  constructor(private modalWindowService: ModalWindowService) {}
+  ngOnInit(): void {
+    this.modalWindowService.visible.subscribe((result: boolean) => {
+      console.log(result);
+      this.cancel();
+    });
+
+    setTimeout(() => {
```

Comment:
Why do we need setTimeout here?

Reply:
> Why do we need setTimeout here?

because otherwise subscribe isn't watch the data. only log, without assign data to variable